corehq/apps/app_manager/suite_xml/case_tile_templates/one_one_two.xml

```diff
@@ -125,7 +125,7 @@
     </header>
     <template form="address" width="0">
       <text>
-        <xpath function="{map[xpath_function]}">
+        <xpath function="concat({map[xpath_function]}, if(here(), '', ''))"> <!-- Without the here() call sessionStorage.locationLat will not be populated -->
```

Comment:
@orangejenny I had to move the `here()` call into the template xpath to make it work correctly. Is there a better way to swallow its output than this if/concat?

Reply:
This seems reasonable. You could do something like `cond(here() and false(), '', {map[xpath_function]})`but I don't think that's necessarily clearer.

All CC xpath function: https://wiki.commcarehq.org/display/commcarepublic/CommCare+Functions